packages/react/src/incentives.ts

```diff
@@ -0,0 +1,60 @@
+import type {
+  MeritClaimRewardsRequest,
+  MeritClaimRewardsTransaction,
+} from '@aave/graphql';
+import { MeritClaimRewardsQuery } from '@aave/graphql';
+import type {
+  ReadResult,
+  Suspendable,
+  SuspendableResult,
+  SuspenseResult,
+} from './helpers';
+import { useSuspendableQuery } from './helpers';
+
+export type UseMeritClaimArgs = MeritClaimRewardsRequest;
+
+/**
+ * Fetch Merit claim rewards transaction for a user.
+ *
+ * This signature supports React Suspense:
+ *
+ * ```tsx
+ * const { data } = useMeritClaim({
+ *   user: evmAddress('0x742d35cc…'),
+ *   chainId: chainId(1),
+ *   suspense: true,
+ * });
+ * ```
+ */
+export function useMeritClaim(
+  args: UseMeritClaimArgs & Suspendable,
+): SuspenseResult<MeritClaimRewardsTransaction | null>;
+
+/**
+ * Fetch Merit claim rewards transaction for a user.
+ *
+ * ```tsx
+ * const { data, loading } = useMeritClaim({
+ *   user: evmAddress('0x742d35cc…'),
+ *   chainId: chainId(1),
+ * });
+ * ```
+ */
+export function useMeritClaim(
+  args: UseMeritClaimArgs,
+): ReadResult<MeritClaimRewardsTransaction | null>;
+
+export function useMeritClaim({
```

Comment:
any reason to have change the terminology here (from 'merit claim rewards' to 'merit claim')?

next to 'use' it feels something different: 'use merit claim'